docs/integration_tests.md

```diff
@@ -145,125 +123,69 @@ In case you plan to use Minishift, the following steps are necessary to get Kept
    Error from server (InternalError): Internal error occurred: unexpected response: 400
    ```
    and retry again.
-1. Set policies/permissions:
+5. Set policies/permissions:
    ```console
    oc adm policy --as system:admin add-cluster-role-to-user cluster-admin admin
    oc adm policy  add-cluster-role-to-user cluster-admin system:serviceaccount:default:default
    oc adm policy  add-cluster-role-to-user cluster-admin system:serviceaccount:kube-system:default
    ```
-1. Note down the Minishift server URL printed by `oc status` (e.g., `https://192.168.99.101:8443`)
-
+6. Note down the Minishift server URL printed by `oc status` (e.g., `https://192.168.99.101:8443`)
 
-## Run the integration tests locally
-
-### Run the quality-gates integration test (with dynatrace-sli-service) locally
+### Run the full installation of Integration Tests locally
 
 Pre-requisites:
 
 * A K8s cluster (see above)
-  * 4 GB of free memory (less is possible but you will notice a significant slowdown)
-  * at least 2 CPU cores on your system (less is possible but you will notice a significant slowdown)
+  * 12 GB of free memory (less is possible but you will notice a significant slowdown)
+  * at least 6 CPU cores on your system (less is possible but you will notice a significant slowdown)
 * `kubectl`
-* `keptn` CLI
 * A local copy of this repository (keptn/keptn)
 * Linux, Mac OS or Linux Subsystem for Windows
 * Bash
-* Dynatrace Tenant with a service that has the tags `FrontEnd` and `testdeploy`
-* Dynatrace API Token that is allowed to read metrics from the Dynatrace Tenant
 * working internet connection
 
-
 1. Setup your Kubernetes cluster (see above)
-1. `keptn` CLI available
-1. Verify `kubectl` is configured to the right cluster:
+2. Install `keptn` CLI:
    ```console
-   kubectl get nodes
+   curl -sL https://get.keptn.sh | KEPTN_VERSION=0.12.0 bash
```

Comment:
Without the `KEPTN_VERSION` env var, the script downloads and installs the latest version. 
```suggestion
   curl -sL https://get.keptn.sh | bash
```